src/Klarna.Checkout/README.md

```diff
@@ -318,6 +318,29 @@ When a payment needs an additional review, the payment in EPiServer is set to th
 The KlarnaPaymentGateway save notes about payment updates at the order.
 ![Order notes](/docs/screenshots/order-notes.PNG?raw=true "Order notes")
 
+</details>
+<details>
+  <summary>External Payment Methods & External Checkout (click to expand)</summary>
+  
+Klarna Checkout offers a wide variety of payment methods to cover the main needs of consumers in all markets, which all are included with a simple, single integration.
```

Comment:
```suggestion
Klarna Checkout offers a wide variety of payment methods to cover the primary needs of consumers in all markets, which all are included with simple, single integration.
```
Suggestions from Gramamrly:
![image](https://user-images.githubusercontent.com/646298/70596943-be2d6080-1bef-11ea-87f1-4a910d4b79f3.png)

![image](https://user-images.githubusercontent.com/646298/70596926-b2da3500-1bef-11ea-8526-d81f63822c8a.png)

Reply:
This was copied from Klarna's documentation. I wanted to keep it the same as over there :)